src/Centreon/Infrastructure/Monitoring/ResourceRepositoryRDB.php

```diff
@@ -413,6 +413,18 @@ public function findResources(ResourceFilter $filter): array
      */
     private function hasServiceSearch(): bool
     {
+        $search = $this->sqlRequestTranslator->getRequestParameters()->getSearch();
+
+        if (!$search) {
```

Comment:
```suggestion
        if (empty($search)) {
```

Reply:
which case do you want to cover?

Reply:
getSearch returns an array, so it's a better practice to use empty instead of "not" comparaison